src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -348,16 +377,25 @@ exports.createFiberFromText = function(
 function createFiberFromElementType(
   type: mixed,
   key: null | string,
+  internalContextTag: TypeOfInternalContext,
   debugOwner: null | Fiber | ReactInstance,
 ): Fiber {
+  if (
+    ReactFeatureFlags.enableAsyncSubtreeAPI &&
+    type != null &&
+    (type: any).unstable_asyncUpdates === true
```

Comment:
I realize that you'll have to mutate the flag to make that happen. That's fine because the children of this fiber won't be created until after that.